src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -852,4 +853,15 @@ public function testMultipleLocationsWithSubDirectories()
         $this->assertIterator($expected, $finder);
         $this->assertIteratorInForeach($expected, $finder);
     }
+
+    public function testShellCommand()
+    {
+        $cmd = Command::create()->add('--force');
+        $this->assertSame('--force', $cmd->join());
```

Comment:
@xabbuh Can you reply to my comment? I'd be happy to refactor the tests, but I need guidance on what effect are you interested in.